viroverse.conf

```diff
@@ -0,0 +1,80 @@
+# Common configuration file for Viroverse. If you want to override the defaults,
+# add them into a local file, viroverse_local.conf
+#
+# This file as well as viroverse_local.conf are in Config::General format
+# (similar to ApacheConf) but any format supported by Perl's Config::Any
+# interpreter will work, including JSON, YAML, or a Perl script, depending on
+# the requirements of the local installation.
+#
+# Many of the defaults are configured at runtime; those settings are documented
+# here but are commented out.
+
+### NODE INFORMATION
+
+# Instance name of the Viroverse server; defaults to $USERNAME-$HOSTNAME
+#instance_name = vverse-localhost
+
+# Whether to run in debug mode or not; defaults to the value of the $VVDEBUG
+# environment variable, or 1 if unset
+#debug = 1
+
+### CONTACT INFORMATION
+
+# The friendly sysadmin name
+help_name = Viroverse Adminiatrator
+
+# The email address to provide for support; defaults to $USERNAME@$HOSTNAME
+#help_email = help@viroverse.example.edu
+
+# The email address to send error reports to; defaults to $USERNAME@$HOSTNAME
+#error_email = admins@viroverse.example.edu
+
+### STORAGE
+
+# Directory for local file storage; defaults to var/storage under the Viroverse
+# installation directory
+#storage = $ENV{VIROVERSE_ROOT}/var/storage
+
+# Database connection information
+dsn = dbi:Pg:host=127.0.0.1;dbname=viroverse;port=5432
+
+# Read-only user
+# Note: for best security practices, use pg_hba.conf to set system-level user
+# authorization rather than using passwords; in any case, don't check passwords
+# into source control.
+read_only_user = viroverse_r
+read_only_pw =
+
+# Read-write user
+read_write_user = viroverse_w
+read_write_pw =
+
+### VIROBLAST
+
+# These defaults won't work unless var/viroblast is a checkout of
+# viroverse-viroblast.git (or a symlink to one)
```

Comment:
We should replace this detail with a proper link to obtaining Viroblast (which for whatever reason is through [CoMotion](https://els.comotion.uw.edu/express_license_technologies/viroblast)).

Reply:
k, I had no idea :)

Reply:
No reason you should've!